### PR TITLE
welcome to Ben10 universe

### PR DESCRIPTION
**Pull request by Gawasinspire**
welcome to ben10 universe :)

### CODE REVIEW

File: _haikus/omnitrix_10.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: haiku
+title: Omnitrix
+author: Ben
+---
+
+
+Ben10 is super cool<br>
```
Comment (ltagliaferri) [inline _haikus/omnitrix_10.md]: Not sure about this one, is `Ben10` one syllable? How is it pronounced?

Comment (Gawasinspire) [reply _haikus/omnitrix_10.md]: it is also 1, totally 5
https://syllablecounter.net/count
https://www.howmanysyllables.com/syllable_counter/

Comment (ltagliaferri) [reply _haikus/omnitrix_10.md]: How do you pronounce it?

**Comment by ltagliaferri** [comment]
Thank you for opening this pull request. The changes requested have not been addressed over the course of a few days, so we are closing this PR due to inactivity and are marking it as `invalid`. If you would like to try again, please open a new pull request.

**Pull request closed without merging**